crates/bitwarden/src/client/client.rs

```diff
@@ -222,6 +222,20 @@ impl Client {
         Ok(self.encryption_settings.as_ref().unwrap())
     }
 
+    #[cfg(feature = "mobile")]
+    pub(crate) fn initialize_user_crypto_decrypted_key(
+        &mut self,
+        decrypted_user_key: &str,
+        private_key: EncString,
+    ) -> Result<&EncryptionSettings> {
+        let user_key = decrypted_user_key.parse::<SymmetricCryptoKey>()?;
+        self.encryption_settings = Some(EncryptionSettings::new_decrypted_key(
+            user_key,
+            private_key,
+        )?);
+        Ok(self.encryption_settings.as_ref().unwrap())
```

Comment:
Do we need to unwrap this instead of properly handling the error?

Reply:
The value is initialized on the previous line, so it should never happen, but at least using `.expect("Value was initialized on the previous line")` would make that more clear